test/Password.ui.ts

```diff
@@ -250,10 +253,13 @@ describe('Password management', () => {
         )
         .click();
 
+      await this.driver.sleep(DEFAULT_ANIMATION_DELAY);
```

Comment:
Can we wait for `until.elementLocated` first and then wait for `until.elementIsVisible` without `sleep`?

for example:
```javascript
await this.driver
  .wait(
    until.elementIsVisible(
      this.driver.wait(
        until.elementLocated(
          By.css('[data-testid="modalPassword"]')
        ),
        this.wait
      )
    ),
    this.wait
  )
  .getText()
).matches(/Password changed/i);
 ```